Improving hand documentation (#97)
shadow-robot/sr_dexterous_hand_documentation: docs/user_guide/1_laptop_box.rst

Laptop box contents 
-------------------------------------------------

> This is an extra box additional to the hand(s) peli case(s). It will only be 1 per bimanual system..

Reviewer: ```suggestion
This is an extra box, additional to the hand(s) peli case(s). There will be 1 per bimanual system.
```
